src/Dapr.Client/GRPCStatusInfo.cs

```diff
@@ -0,0 +1,48 @@
+﻿// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+// ------------------------------------------------------------
+namespace Dapr.Client
+{
+    using Dapr.Client.Http;
+    using System;
+    using System.Net;
+
+    /// <summary>
+    /// This class is only needed if the app you are calling app is listening on gRPC.
+    /// It contains propertes that represent status info that may be populated for an gRPC response.
+    /// </summary>
+    public class GRPCStatusInfo
+    {
+        /// <summary>
+        /// The constructor.
+        /// </summary>
+        public GRPCStatusInfo(Grpc.Core.StatusCode statusCode, string message, int? innerHttpStatusCode = default, string InnerHttpErrorMessage = default)
+        {
+            this.GrpcStatusCode = statusCode;
+            this.GrpcErrorMessage = message;
+            this.InnerHttpStatusCode = innerHttpStatusCode;
+            this.InnerHttpErrorMessage = InnerHttpErrorMessage;
```

Comment:
What's the meaning of these two?

Reply:
in the case of grpc to http, the innerHttpStatusCode and error message are the http status code and error message returned by the HTTP server. the grpcstatuscode and grpcerrormessage has the values returned by Dapr which are converted values from http to grpc